app/src/main/java/com/infomaniak/mail/utils/WebViewUtils.kt

```diff
@@ -100,7 +117,11 @@ class WebViewUtils(context: Context) {
 
     companion object {
         private const val DARK_BACKGROUND_STYLE_ID = "dark_background_style"
-        val jsBridge = JavascriptBridge() // TODO: Avoid excessive memory consumption with injection
+        var jsBridge: JavascriptBridge? = null // TODO: Avoid excessive memory consumption with injection
```

Comment:
Do we need to make it nullable if we're gonna init it anyway right after? Wouldn't `lateinit` be enough to avoid writting a line of code of `?.let` on a value that will always be set anyway?